MuonFlux-HyperK-ThetaPhi.dat

```diff
@@ -0,0 +1,18000 @@
+0 0 0.005 0 0.01 1 0 2 1.45271e-12 452.074
```

Comment:
I really like this suggestion. If there are other inputs that other groups are creating it would be useful to have a data folder. I will put the file in there for now.